shared/common-adapters/avatar.tsx

```diff
@@ -108,13 +107,9 @@ const ConnectedAvatar = Container.connect(
     _httpSrvAddress: state.config.httpSrvAddress,
     _httpSrvToken: state.config.httpSrvToken,
     blocked:
-      (
-        (state.tracker2 &&
-          state.tracker2.usernameToDetails &&
-          Tracker2Constants.getDetails(state, ownProps.username || ownProps.teamname || '')) || {
-          blocked: false,
-        }
-      ).blocked || false,
+      state.users &&
+      state.users.blockMap &&
+      state.users.blockMap.get(ownProps.username || ownProps.teamname || '')?.chatBlocked,
```

Comment:
`state.users?.blockMap?.get(...)?.chatBlocked ?? false` would be a little more concise here